src/main/java/cz/muni/fi/DriverDao.java

```diff
@@ -0,0 +1,77 @@
+package cz.muni.fi;
+
+import cz.muni.fi.entities.Driver;
+
+import java.util.List;
+
+/**
+ * @author Richard Hrmo
+ */
+public interface DriverDao{
+    /**
+     * List all drivers
+     *
+     * @param None
```

Comment:
tag `@param` is not needed when your method does not take anything but nevermind now